### PR TITLE
homepage and etherscan api working

### PR DESCRIPTION
**Pull request by samueltrahan**
I think there are some things we can take out of the code.  I don't remember what we had talked about taking out, but I will do that.

### CODE REVIEW

**Comment by samueltrahan** [comment]
Fixed the merge conflicts

File: src/App.tsx [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
-      const { data } = await axios.get(
-        `http://api.etherscan.io/api?module=account&action=txlist&address=${privateKey}&startblock=0&endblock=99999999&sort=asc&apikey=${etherscanAPI}`,
-      );
-      setAccount(data);
-    })();
-  }, [privateKey]);
+    axios.get(`http://api.etherscan.io/api?module=account&action=txlist&address=${privateKey}&startblock=0&endblock=99999999&sort=asc&apikey=${etherscanAPI}`)
+    .then(res => setAccount(res.data))
+  }, [privateKey])
+
```
Comment (jpangelle) [inline src/App.tsx:29]: Could just leave this part. it is using `async/await` instead of `then`

File: src/App.tsx [hunk 2]
```diff
@@ -1,43 +1,55 @@
...
 import SetupPage from './pages/SetupPage/SetupPage';
 import GeneratePage from './pages/GenerateWalletPage/Generate';
 import ExistingPage from './pages/ExistingWalletPage/Existing';
 import HomePage from './pages/HomePage/HomePage';
-import './App.css';
+
+
+const Props = {
+  setPrivateKey: 'string'
+}
```
Comment (jpangelle) [inline src/App.tsx:14]: remove this part

File: src/App.tsx [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+
 
 const etherscanAPI = process.env.REACT_APP_ETHERSCAN_API;
 
 function App() {
   const [account, setAccount] = useState([]);
-  const [privateKey, setPrivateKey] = useState('');
+
+  const [privateKey, setPrivateKey] = useState('')
+
```
Comment (jpangelle) [inline src/App.tsx:23]: looks like there is no real changes here, can remove this change

File: src/App.tsx [hunk 4]
```diff
@@ -1,43 +1,55 @@
...
+        </Route>
+         <Route exact path='/existing' render={() => 
+        <ExistingPage setPrivateKey={setPrivateKey}/>
+        }>
+         </Route>
+         <Route exact path="/homepage" render={() => 
+           <HomePage privateKey={privateKey}/>
+         }>
+         </Route>
+
```
Comment (jpangelle) [inline src/App.tsx:52]: Does not look there are any changes here either. Just formatting changes. you can ignore these changes too

File: src/index.tsx [hunk 5]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
-import './index.css';
+import { BrowserRouter as Router} from 'react-router-dom';
```
Comment (jpangelle) [inline src/index.tsx:5]: This is already imported on line 3. You can remove this change.

File: src/pages/ExistingWalletPage/Existing.tsx [hunk 6]
```diff
@@ -1,17 +1,45 @@
...
 type Props = {
   setPrivateKey: (privateKey: string) => void;
-};
+}
+
+
+
+
+export default function Existing({setPrivateKey}: Props) {
+  const [recoveryProcess, setRecoveryProcess] = useState<string | undefined>();
```
Comment (jpangelle) [inline src/pages/ExistingWalletPage/Existing.tsx:15]: These two lines are already below.

File: src/pages/ExistingWalletPage/Existing.tsx [hunk 7]
```diff
@@ -1,17 +1,45 @@
...
+    const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+      setRecoveryProcess(event.target.value)
+      
+    }
+
+    const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+      event.preventDefault();
+      setPrivateKey(event.target.value)
+     
+    }
```
Comment (jpangelle) [inline src/pages/ExistingWalletPage/Existing.tsx:41]: This part is already below.

**Pull request closed without merging**